### PR TITLE
feat(User): add openhexa_legacy and workspaces features to root

### PR DESCRIPTION
**Pull request by cheikhgwane**
Add openhexa_legacy and workspaces feature flag to root user when installing fixtures.

## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- add features in fixtures
- use natural key in fixtures for user model 

## How/what to test

Have a fresh install of openhexa-app and check if root user have access to legacy and workspaces part.

## Screenshots / screencast

If relevant, please add some screenshots or a short video.

### CODE REVIEW

**Comment by cheikhgwane** [comment]
@pvanliefland  it should be fine now

File: hexa/user_management/models.py [hunk 1]
```diff
@@ -53,6 +53,9 @@ def create_superuser(self, email=None, password=None, **extra_fields):
             raise ValueError(_("Superuser must have is_superuser=True."))
         return self.create_user(email, password, **extra_fields)
 
+    def get_by_natural_key(self, email: str):
```
Comment (pvanliefland) [inline hexa/user_management/models.py:56]: Are you sure this override is necessary?

The code from the BaseUserManager in Django is

```python
def get_by_natural_key(self, username):
    return self.get(**{self.model.USERNAME_FIELD: username})
```

It should work for us now?

Comment (cheikhgwane) [reply hexa/user_management/models.py:56]: Oh. Yes, you're right. I've missed this one

File: hexa/user_management/models.py [hunk 2]
```diff
@@ -280,13 +283,20 @@ def delete_if_has_perm(self, principal: User):
         return super().delete()
 
 
+class FeatureManager(models.Manager):
```
[the remark this answers is not in the document]
Comment (cheikhgwane) [reply hexa/user_management/models.py:286]: In order to use the get_natural_key method we need to use a model Manager. (see https://docs.djangoproject.com/en/4.2/topics/serialization/#deserialization-of-natural-keys)

[the remark this answers is not in the document]
Comment (cheikhgwane) [reply hexa/user_management/models.py:286]: Not reallly. We can use the pk field.

**Comment by pvanliefland** [comment]
@cheikhgwane the tests are failing

**Comment by cheikhgwane** [comment]
@pvanliefland according to the [doc](https://docs.djangoproject.com/en/4.2/topics/serialization/#serialization-of-natural-keys) we need to add a get_by_natural_key on the model manager and natural_key method on model.

**Review by pvanliefland — APPROVED** [review]
LGTM!

**Pull request merged**